modules/statics/src/coins.ts

```diff
@@ -386,7 +386,8 @@ const ZKETH_FEATURES = [
 ];
 const BERA_FEATURES = [
   ...ETH_FEATURES,
-  CoinFeature.MULTISIG_COLD,
+  CoinFeature.TSS,
```

Comment:
Does this mean that bera started out as multisig and we are switching to TSS before rolling it out?

Reply:
yes, we are switching to TSS. this decision is made before moving to mainnet